activesupport/lib/active_support/cache/strategy/local_cache.rb

```diff
@@ -131,17 +131,20 @@ def read_serialized_entry(key, raw: false, **options)
             end
           end
 
-          def read_multi_entries(keys, **options)
+          def read_multi_entries(names, **options)
             return super unless local_cache
 
-            local_entries = local_cache.read_multi_entries(keys)
+            keys_to_names = names.index_by { |name| normalize_key(name, options) }
+
+            local_entries = local_cache.read_multi_entries(keys_to_names.keys)
+            local_entries.transform_keys! { |key| keys_to_names[key] }
             local_entries.transform_values! do |payload|
-              deserialize_entry(payload)&.value
+              deserialize_entry(payload, raw: options[:raw])&.value
```

Comment:
I don't think `LocalCache` should have knowledge of the `:raw` option.  I _think_ we might be better off just passing `options` here:

```suggestion
              deserialize_entry(payload, **options)&.value
```

Reply:
that sounds like it makes sense... I'll test to be sure :-)